NeedABreak/SettingsWindow.xaml.cs

```diff
@@ -163,5 +165,13 @@ private void CloseButton_Click(object sender, RoutedEventArgs e)
         {
             Close();
         }
+        private void LockNow(object sender, RoutedEventArgs e)
+        {
+            //System.Diagnostics.Process.Start("rundll32.exe", "user32.dll LockWorkStation");
+
```

Comment:
Please remove this commented out line.

Reply:
I just realized that you have added the button on the wrong Window. The "lock now" button is intended to be on the MainWindow which shows a countdown before locking. So the user will have 3 choices :
* Postpone
* Lock now
* Cancel

The "Lock now" button will permit the user to lock the screen immediatly and so he won't have to wait until the end of the countdown.

To easily show the MainWindow (and don't wait 45 minutes) you can set the static variable Delay to a low value in the App class.

Reply:
Oh I see. I will review the code and make the required changes.